onnx/reference/ops/op_average_pool.py

```diff
@@ -27,3 +28,89 @@ def _run(  # type: ignore
             pads=pads,
             strides=strides,
         )
+
+
+class AveragePool_7(CommonPool):
+    def _run(  # type: ignore
+        self,
+        x,
+        auto_pad=None,
+        ceil_mode=None,
+        kernel_shape=None,
+        pads=None,
+        strides=None,
+        count_include_pad=None,
+    ):
+        return CommonPool._run(
+            self,
+            "AVG",
+            count_include_pad,
+            x,
+            auto_pad=auto_pad,
+            ceil_mode=ceil_mode,
+            dilations=None,
+            kernel_shape=kernel_shape,
+            pads=pads,
+            strides=strides,
+        )
+
+
+class AveragePool_11(CommonPool):
+    def _run(  # type: ignore
+        self,
+        x,
+        auto_pad=None,
+        ceil_mode=None,
+        kernel_shape=None,
+        pads=None,
+        strides=None,
+        count_include_pad=None,
+    ):
+        return CommonPool._run(
+            self,
+            "AVG",
+            count_include_pad,
+            x,
+            auto_pad=auto_pad,
+            ceil_mode=ceil_mode,
+            dilations=None,
+            kernel_shape=kernel_shape,
+            pads=pads,
+            strides=strides,
+        )
+
+
+class AveragePool_19(CommonPool):
+    def _run(  # type: ignore
+        self,
+        x,
+        auto_pad=None,
+        ceil_mode=None,
+        dilations=None,
+        kernel_shape=None,
+        pads=None,
+        strides=None,
+        count_include_pad=None,
+    ):
+        return CommonPool._run(
+            self,
+            "AVG",
+            count_include_pad,
+            x,
+            auto_pad=auto_pad,
+            ceil_mode=ceil_mode,
+            dilations=dilations,
+            kernel_shape=kernel_shape,
+            pads=pads,
+            strides=strides,
+        )
+
+
+if onnx_opset_version() > 11:
```

Comment:
It should be `>= 19`. AveragePool - 11 is used from opset 11 to 18.

Reply:
Similar points for the checks below, I think?

Reply:
Yes. I removed these lines in #4800 to make it simpler.